libs/designsystem/accordion/src/accordion-item.component.scss

```diff
@@ -41,7 +41,7 @@ $padding: utils.size('s');
 }
 
 .content-body {
-  padding: 0 $padding $padding $padding;
+  padding: 0 $padding $padding;
```

Comment:
This happens due to stylelint/scss being bumped from 6.2.1 -> 6.3.0 under the hood.
From the stylelint/scss changelog: Fixed: no-global-function-names report errors inside at-rules (https://github.com/stylelint-scss/stylelint-scss/pull/988).